### PR TITLE
[Excel] (add-in commands) re-preview enable/disable and GA RibbonApi …

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
…Req Set
@mscharlock They decided at the last minute that the enable/disable APIs would stay in preview a while longer, so I'm reversing that change. But the Req Set **is** releasing, so I added an article for that.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[702d0df](https://github.com/OfficeDev/office-js-docs-pr/commits/702d0df23ddc8c00f0b228089f65a543d277c0d9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/disable-add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/design/disable-add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/ribbon-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/reference/requirement-sets/ribbon-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005282143295095-1896%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=d3kACu3hbOpYK2a0O98XB7S5Rly5RVW5lcyTxElILFo%3D&st=2020-05-28T21%3A39%3A51Z&se=2020-06-28T21%3A44%3A51Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[05aa53a](https://github.com/OfficeDev/office-js-docs-pr/commits/05aa53a11bf5d421acaa8c688a866bbe553b9dca)_: 

:clock10: **Pending**: waiting for processors ([6](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 6 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[05aa53a](https://github.com/OfficeDev/office-js-docs-pr/commits/05aa53a11bf5d421acaa8c688a866bbe553b9dca)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/disable-add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/design/disable-add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/ribbon-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/reference/requirement-sets/ribbon-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005291855039923-1896%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=oAbl5AbI9KkwVn8%2FOtiuRvlWVZJNnC05oSDuOxnaSK0%3D&st=2020-05-29T18%3A51%3A42Z&se=2020-06-29T18%3A56%3A42Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/design/disable-add-in-commands.md [hunk 1]
```diff
@@ -16,11 +16,18 @@ You can also specify whether the command is enabled or disabled when the Office
...
+
+> [!NOTE]
+> [!INCLUDE [Information about using preview APIs](../includes/using-preview-apis.md)]
+
 ## Rules and gotchas
 
 ### Single-line ribbon in Office on the web
 
-In Office on the web, the APIs and manifest markup described in this article only affect the single-line ribbon. They have no effect on the multiline ribbon. They affect both ribbons for desktop Office. For more information about the two ribbons, see [Use the simplified ribbon](https://support.office.com/article/Use-the-Simplified-Ribbon-44bef9c3-295d-4092-b7f0-f471fa629a98).
+In Office on the web, the APIs and manifest markup described in this article only affect the single-line ribbon. They have no effect on the multiline ribbon. They affect both ribbons for desktop Office. For more information about the two ribbons, see [The new look of Office - Simplified Ribbon](https://support.microsoft.com/en-us/office/the-new-look-of-office-a6cdf19a-b2bd-4be1-9515-d74a37aa59bf).
```
Comment (mscharlock) [inline docs/design/disable-add-in-commands.md:30]: I would take out the en-us part of the URL

File: docs/reference/requirement-sets/ribbon-api-requirement-sets.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+|  Requirement set  | Office 2013 on Windows<br>(one-time purchase) | Office 2016 or later on Windows<br>(one-time purchase)   | Office on Windows\*<br>(connected to Office 365 subscription) |  Office on iPad<br>(connected to Office 365 subscription)  |  Office on Mac<br>(connected to Office 365 subscription)  | Office on the web  |  Office Online Server  |
+|:-----|-----|:-----|:-----|:-----|:-----|:-----|:-----|
+| RibbonApi 1.1  | N/A | N/A | Version 2002 (Build 12527.20264) or later | 16.38 or later | N/A | February 2020 | N/A|
+
+> **&#42;** During the preview phase, the Ribbon API is supported only on Excel and it requires Office 365 (the subscription version of Office). You should use the latest monthly version and build from the Insiders channel. You need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1). Please note that when a build graduates to the production semi-annual channel, support for preview features, including the Ribbon API, is turned off for that build.
```
Comment (mscharlock) [inline docs/reference/requirement-sets/ribbon-api-requirement-sets.md:21]: Maybe make a note vs. the *?

Comment (Rick-Kirkham) [reply docs/reference/requirement-sets/ribbon-api-requirement-sets.md:21]: Our pattern in the Req Set articles is to use asterisks in the tables. See for example, the IdentityApi and AddinCommands Req Set articles. So, for the sake of consistency, I'd like to keep  the asterisk here. We can revisit whether we should be using notes or asterisks someday, but that would be out-of-scope for this PR.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d1fdde6](https://github.com/OfficeDev/office-js-docs-pr/commits/d1fdde6cd961b6ae220626bb3bae1cee8255ecbc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/disable-add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/design/disable-add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/ribbon-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/reference/requirement-sets/ribbon-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005292119245916-1896%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=OfbA8i9Z9OeNofsN5q%2FW63mOjmRZ1G4MoI0fjx1kisU%3D&st=2020-05-29T21%3A16%3A06Z&se=2020-06-29T21%3A21%3A06Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/design/disable-add-in-commands.md [hunk 3]
```diff
@@ -16,11 +16,18 @@ You can also specify whether the command is enabled or disabled when the Office
...
+
+> [!NOTE]
+> [!INCLUDE [Information about using preview APIs](../includes/using-preview-apis.md)]
+
 ## Rules and gotchas
 
 ### Single-line ribbon in Office on the web
 
-In Office on the web, the APIs and manifest markup described in this article only affect the single-line ribbon. They have no effect on the multiline ribbon. They affect both ribbons for desktop Office. For more information about the two ribbons, see [Use the simplified ribbon](https://support.office.com/article/Use-the-Simplified-Ribbon-44bef9c3-295d-4092-b7f0-f471fa629a98).
+In Office on the web, the APIs and manifest markup described in this article only affect the single-line ribbon. They have no effect on the multiline ribbon. They affect both ribbons for desktop Office. For more information about the two ribbons, see [The new look of Office - Simplified Ribbon](https://support.microsoft.com/office/the-new-look-of-office-a6cdf19a-b2bd-4be1-9515-d74a37aa59bf).
```
Comment (ElizabethSamuel-MSFT) [inline docs/design/disable-add-in-commands.md:30]: Remember to use the guidance in our OneNote for how to make the support.microsoft.com locale-agnostic. So in this case, the link should likely be updated to something like the following.
```suggestion
In Office on the web, the APIs and manifest markup described in this article only affect the single-line ribbon. They have no effect on the multiline ribbon. They affect both ribbons for desktop Office. For more information about the two ribbons, see [The new look of Office - Simplified Ribbon](https://support.microsoft.com/office/a6cdf19a-b2bd-4be1-9515-d74a37aa59bf).
```

Comment (mscharlock) [reply docs/design/disable-add-in-commands.md:30]: Thanks for the email heads up on this, I must have missed this note to do this! :)

**Comment by o365devx** [comment]
Docs Build status updates of commit _[7057603](https://github.com/OfficeDev/office-js-docs-pr/commits/70576039721982d6da24728558853652a1e31cd9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/disable-add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/design/disable-add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/ribbon-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/reference/requirement-sets/ribbon-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005292202299778-1896%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=v9CDWRaBKXslJqkb1YoPHq6LOBaIn2ozqsowe2XpjvE%3D&st=2020-05-29T21%3A59%3A16Z&se=2020-06-29T22%3A04%3A16Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8cf3187](https://github.com/OfficeDev/office-js-docs-pr/commits/8cf318787f2f74f6136fb089e6bc48693d16f8a8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/disable-add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/design/disable-add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/ribbon-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/reference/requirement-sets/ribbon-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005292334074053-1896%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2wwSWgDjSe4%2FooHIt0TdOeZi1Zai6gBM1XaLLsCZ7tg%3D&st=2020-05-29T23%3A30%3A59Z&se=2020-06-29T23%3A35%3A59Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[7a077d9](https://github.com/OfficeDev/office-js-docs-pr/commits/7a077d9918ffe5ca0ec8aee989dc30fc61825e3f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/disable-add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/design/disable-add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/ribbon-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/reference/requirement-sets/ribbon-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/enable-disable-commands/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005292335018873-1896%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=zkXBdnkf4HOLY%2FxXFP1cOnP1ieqUpMNLef0wy1i%2B2tY%3D&st=2020-05-29T23%3A32%3A47Z&se=2020-06-29T23%3A37%3A47Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**